server/pulp/server/db/model/__init__.py

```diff
@@ -307,3 +312,132 @@ def post_save(cls, sender, document, **kwargs):
 
 
 signals.post_save.connect(TaskStatus.post_save, sender=TaskStatus)
+
+
+class ContentUnit(Document):
+    """
+    The base class for all content units.
+
+    All unit inheriting from this class should override the unit_type_id and _ns to ensure
+    that they are populated properly
+
+
+    :ivar id: string representation of content unit id
+    :type id: mongoengine.StringField
+    :ivar last_updated: integer representation of last time this unit was updated (mer
```

Comment:
the type should be clear from the next line. `integer` can be removed. Suggest rewritten to:  "last time this unit was updated" This also gets rid of the `(mer` at the end. I don't know what this is.